scripts/prepare_automatic_backups.py

```diff
@@ -34,6 +34,14 @@
 
 
 def generate_backup_url(cloud_storage_bucket_name, module_class_names):
+    """Genrating backup url of update content.
+    
+    Args:
+      cloud_storage_bucket_name type=string  . name of cloud storage bucket name.
+      module_class_names type=list . list of contaning modules class names.
+      
+    Returns type=string . returning url of backup.
```

Comment:
This format for `Args:` and `Returns:` is wrong. Please refer to this [wiki](https://github.com/oppia/oppia/wiki/Coding-style-guide#python) page for correct format. Also, indent `cloud_storage_bucket_name` by 4 spaces and not by 2. Similarly for all other args. Don't write `type=string`. Instead please write `str` only.

Reply:
resolved it.